### PR TITLE
Add new function to display arrow

### PR DESCRIPTION
**Pull request by Nandite**
@VictorLamoine 
We added this function for making the publish of arrow easier. You just have to have to give as parameters the start point and the end point of your arrow. I think it is more convenient than using a affine3d or geometry_msgs Pose when you just want to display an arrow. What do you think about that ?

### CODE REVIEW

File: src/rviz_visual_tools.cpp [hunk 1]
```diff
@@ -1331,6 +1331,44 @@ bool RvizVisualTools::publishArrow(const geometry_msgs::PoseStamped &pose, color
...
+  // Set the frame ID and timestamp.
+  arrow_marker_.header.stamp = ros::Time::now();
+  arrow_marker_.header.frame_id = base_frame_;
+
+  if (id == 0)
+    arrow_marker_.id++;
+  else
+    arrow_marker_.id = id;
+
+  geometry_msgs::Point start;
```
Comment (davetcoleman) [inline src/rviz_visual_tools.cpp]: try helper function convertPoint()

File: src/rviz_visual_tools.cpp [hunk 2]
```diff
@@ -1331,6 +1331,44 @@ bool RvizVisualTools::publishArrow(const geometry_msgs::PoseStamped &pose, color
...
+  start.y = origin[1];
+  start.z = origin[2];
+
+  geometry_msgs::Point end;
+  end.x = end_point[0];
+  end.y = end_point[1];
+  end.z = end_point[2];
+
+  arrow_marker_.points.push_back(start);
+  arrow_marker_.points.push_back(end);
```
Comment (davetcoleman) [inline src/rviz_visual_tools.cpp]: I didn't realize this works, great!

**Comment by davetcoleman** [comment]
why was this closed??

**Comment by Nandite** [comment]
@VictorLamoine We've experienced a bug trying to use that with our project. But when we ran it into your rviz demo it worked well ! So i will make a new pull request as soon as we fix this bug into our work !

**Comment by VictorLamoine** [comment]
I don't know why Nandite closed it.

It was not working but we are working on fixing this :)

**Pull request closed without merging**